### PR TITLE
caja-file-operations: avoid gtk_dialog_add_buttons with stock ids

### PR DESCRIPTION
**Pull request by sc0w**

### CODE REVIEW

File: libcaja-private/caja-file-operations.c [hunk 1]
```diff
@@ -1112,7 +1132,13 @@ do_run_simple_dialog (gpointer _data)
 			continue;
 		}
 
-		gtk_dialog_add_button (GTK_DIALOG (dialog), button_title, response_id);
+		if (button_title == CANCEL)
```
Comment (monsta) [inline libcaja-private/caja-file-operations.c]: Like it was in another PR, here strings are compared in improper way. However, I don't know if `g_strcmp0` could really be used for strings wrapped with `_( ... )` for translation.

**Comment by sc0w** [comment]
@monsta 

oops, yes, fixed.

It seems to work here, you can test it with gtk-inspector, the GtkImage properties: icon-name, stock, and storage-type

**Comment by raveit65** [comment]
Caja has so many dialogs, so which dialog is it?

**Comment by sc0w** [comment]
@raveit65 this PR affects a lot of dialogs, for example:

002caja

001caja

**Comment by sc0w** [comment]
merged: https://github.com/mate-desktop/caja/commit/5f0ad00b5f3c48c01bf3d9444cb186437932e64c

**Pull request closed without merging**